tutorials/vscode.md

```diff
@@ -11,4 +11,60 @@
 This page contains **'community resources' contributed by students**.
 </box>
 
-Coming soon ...
\ No newline at end of file
+## Importing a Gradle project
+
+<box type="tip" seamless>
+
+If the project comes with Gradle support, you will see a build.gradle file in your project root.
+</box>
+
+<box type="warning" seamless>
+
+Install the [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack) extension to enable support for Java extensions.  
+Go to `Extensions` → Search `Extension Pack for Java` → Install.
+</box>
+
+1. Open Visual Studio Code.
+
+2. If you are in the welcome screen, Click Open. Otherwise, click File -> Open.
+   i. Select the project directory, and click Open.
+
+3. Confirm the correct Java version is being used for Gradle, as follows:
+
+    (a) Confirm the project JDK is set to the one you are supposed to use for the project, as given in the panel below:
+
+    i. Open the Command Palette by pressing `Ctrl+Shift+P` (Windows/Linux) or `Cmd+Shift+P` (Mac).
+
```

Comment:
This was a nice touch that personalises to the reader and makes it more convenient!